lib/hypernova/blank_renderer.rb

```diff
@@ -31,4 +31,8 @@ def key
   def name
     job[:name]
   end
+
+  def id
+    SecureRandom.uuid
```

Comment:
I would like to cherry-pick this one. However, I kept failing with the error `fatal: reference is not a tree: e0b8ad7496d418a5b5dfced30290cfcfc3e665db` when I checkout out to your commit. And cherry-pick returns another error `fatal: bad object e0b8ad7496d418a5b5dfced30290cfcfc3e665db`. So I just copy and paste the adjustment to my commit. Perhaps I just missed something?